homeassistant/components/axis/config_flow.py

```diff
@@ -74,9 +74,8 @@ async def async_step_user(
                     password=user_input[CONF_PASSWORD],
                 )
 
-                self.serial = device.vapix.serial_number
-                assert self.serial
-                await self.async_set_unique_id(format_mac(self.serial))
+                self.serial = serial_number = device.vapix.serial_number
```

Comment:
I'm not following this one?

- Please don't assign multiple variables in a single line
- This is working around the original comment that `self.serial` is initialized incorrectly IMHO; thus this didn't address my comment above.

Reply:
I have removed the `self.serial` variable completely, as it is only used once can be passed as an argument instead.